src/components/form/fields/passwordField.jsx

```diff
@@ -170,18 +181,20 @@ export const passwordField = ({ label = 'password', canShowPassword = false, ...
                 strength: 0,
                 enabledRules: listEnabledRules(i18n, passwordPolicy),
                 isTouched: false,
-                isDirty: false
+                isDirty: false,
+                blacklist: [],
             }),
             unbind: (model, { value }) => ({ ...model, password: value }),
-            validate: ({ value, strength, isDirty }, ctx) => {
+            validate: ({ value, strength, isDirty, blacklist }, ctx) => {
                 if (!isDirty && !ctx.isSubmitted) return {};
 
                 const errors = [];
                 if (!value) {
                     errors.push(i18n('validation.required'));
                 }
                 else {
-                    if (strength < passwordPolicy.minStrength) {
+                    const includesBannedWords = blacklist.some(str => value.toLowerCase().includes(str));
+                    if (includesBannedWords || strength < passwordPolicy.minStrength) {
                         errors.push(i18n('validation.password.minStrength'));
                     }
```

Comment:
Message should be different to me. @mehdi-reach5 ?

Reply:
Seen with Mehdi already. Also, no specific i18n was specified in the ticket; it would have to be added at the backend first.